program_intent_engagement/apps/core/models.py

```diff
@@ -36,3 +37,43 @@ def get_full_name(self):
 
     def __str__(self):
         return str(self.get_full_name())
+
+
+class ProgramIntent(TimeStampedModel):
```

Comment:
General question - 

Will this model be the way we are interacting with snowflake generated data vs data that is being written via API calls? 

Are the two data sources being stored in separate databases, or is the management command going to make use of the API to insert data into the store? 

Wondering because if we are creating a kind of data pipeline to go from snowflake to the DB (like in insights), we probably don't want to inherit from `TimeStampedModel`

Reply:
it's a good question that I think we want to look at in MST-1498

we need the ability to make intents directly from the live system, for example when a user says stop showing me this program on my program dashboard because it reminds me of past failures and makes me sad (this would be CERTAIN_NO with reason DEEP_REGRET). 

we also want to load them offline

I think the right answer is probably two different models (and maybe two different DBs)